### ISSUE TITLE
masigpro and deseq 

### ISSUE THREAD

**Issue by BeatrizdeToledo** [ConesaLab/tappAS#27]
Good afternoon,

I performed PacBio iso-seq and after SQANTI3 filtering I quantified transcripts using RSEM-STAR.
I want to compare 3 cell populations and even after reading tappas paper/website and masigpro paper I still have a question regarding the statistics analysis i should use. 

I am using Tappas and I tried to use masigpro to do the analysis of the 3 cell types at the same time, and obtained 2600 genes having DIU.
When I tried to compare Cell 1 x Cell2, applying DESEQ I obtained only 600 genes.

I didn't expect to observe this huge difference.
Given the fact I have only 3 time points, is it possible that masigpro is not ideal for my analysis? 

Do you have any sugestions?

Thank you!
Beatriz

**Comment by BeatrizdeToledo** [comment]
I need to add some additional information.
I had a list of transcripts that were DIU of at least 10% between Cell 1 and Cell2. 
Then I checked among those, which ones were part of genes with q-value <0,05. I had the values obtained with masigpro and deseq.
I then obtained 2600 genes with masigpro and 600 genes with DESEQ.

I would expect more genes in the masigpro because maybe there is no difference between cell 1 and cell 2 but there is a diference on the "time" profile. But is it supposed to be that big?
